docs/modules/ROOT/pages/index.adoc

```diff
@@ -2,13 +2,48 @@
 
 image::syn.png[]
 
-This is the documentation of Project Syn.
+== Introduction
 
-== Components
+> Project Syn is a pre-integrated set of tools to provision, update, backup, observe and react/alert production applications on Kubernetes and in the cloud. It supports DevOps through full self-service and automation using containers, Kubernetes and GitOps.
+
+All code lives on GitHub under https://github.com/projectsyn.
+
+=== What's in for: Dev
+
+The tooling provided by Project Syn enables the user of the platform to consume all the available services in self-service. By fully automating the day-2 operations (see below), all consumed services are automatically "managed". There is no need to use different configuration management tooling, it's all defined at the same place: Kubernetes objects in a Git repository.
+
+* Automated service deployment with https://crossplane.io/[Crossplane].
+* Backup of data with https://k8up.io[K8up] and https://restic.readthedocs.io[Restic].
+* GitOps with https://argoproj.github.io/argo-cd/[Argo CD].
+* Secrets management with https://www.vaultproject.io/[Vault].
+* Monitoring and alerting with Prometheus, Alertmanager and Signalilo.
```

Comment:
Please add links like with the tools mentioned above.

Reply:
Done in da4645d